.github/workflows/gradle.yml

```diff
@@ -7,12 +7,19 @@ env:
 on:
   #ability to start task manually in Web UI
   workflow_dispatch:
+    inputs:
+      debug_mode:
+        description: "Enable or disable tmate session for debug during helm ac tests"
+        default: "true"
```

Comment:
let's default to "false"

Reply:
Hmm, i thought it was on false :) need to revisit the if statement then, since its not working properly.

Reply:
Done